NOTICES.md

```diff
@@ -15,3 +15,30 @@ distributed in its original form without modification.
 
 Please run the command `yarn licenses list` for a full list of all of the
 licenses used by this package's dependencies.
+
+--
```

Comment:
I happened to find the code that was used to create this clustering. We need to give credit where credit is due when re-using code.

Reply:
I agree that this had to be done. Thanks for adding this